### PR TITLE
Devop: Buy/Sell on Moonpay & Simplex

### PR DESCRIPTION
**Pull request by Abyscuit**
### Devop

* \[x] Added entry to ./changelog
* \[x] Add PR label

### CODE REVIEW

**Review by andrewdahn — APPROVED** [review]

File: connections.js [hunk 1]
```diff
@@ -48,6 +48,7 @@ module.exports = [
   'https://registry.walletconnect.org:443/data/wallets.json',
   'wss://*.bridge.walletconnect.org:443',
   'https://buy-staging.moonpay.com:443',
+  'https://api.coingecko.com:443',
```
Comment (gamalielhere) [inline connections.js:51]: can probably remove this

File: src/modules/moon-pay/ModuleMoonPay.vue [hunk 2]
```diff
@@ -10,49 +10,59 @@
...
-          <buy-eth-component
-            :moonpay-handler="moonpayHandler"
-            :close="close"
-            :tab="activeTab"
-            :default-currency="defaltCurrency"
-            :in-wallet="inWallet"
-            @selectedCurrency="setSelectedCurrency"
-          />
-        </div>
+      <div v-if="step == 0">
```
Comment (gamalielhere) [inline src/modules/moon-pay/ModuleMoonPay.vue:13]: let's use strict comparisons `===`

File: src/modules/moon-pay/ModuleMoonPay.vue [hunk 3]
```diff
@@ -10,49 +10,59 @@
...
+              :tab="activeTab"
+              :in-wallet="inWallet"
+              :default-currency="defaltCurrency"
+              @selectedCurrency="setSelectedCurrency"
+            />
+          </template>
+        </mew-tabs>
       </div>
+      <MoonPayBuyProviderComponent
+        v-if="step == 1"
```
Comment (gamalielhere) [inline src/modules/moon-pay/ModuleMoonPay.vue:52]: same here

File: src/modules/moon-pay/components/MoonPayBuyComponent.vue [hunk 4]
```diff
@@ -612,116 +444,87 @@ export default {
...
+        selectedCryptoName: this.selectedCryptoName,
+        plusFeeF: this.plusFeeF,
+        includesFeeText: this.includesFeeText,
+        networkFeeText: this.networkFeeText,
+        dailyLimit: this.dailyLimit,
+        monthlyLimit: this.monthlyLimit,
+        fiatAmount: this.amount
+      };
+      this.checkMoonPayMax();
+      this.$emit('simplexQuote', this.simplexQuote);
```
Comment (gamalielhere) [inline src/modules/moon-pay/components/MoonPayBuyComponent.vue:522]: you can probably change this to 1 emit by passing an array instead `this.$emit(<event>, [...,])`

File: src/modules/moon-pay/components/MoonPaySellComponent.vue [hunk 5]
```diff
@@ -324,9 +378,9 @@ export default {
       },
       deep: true
     },
-    defaultCurrency(e) {
-      this.selectedCurrency = e;
-    },
+    // defaultCurrency(e) {
```
Comment (gamalielhere) [inline src/modules/moon-pay/components/MoonPaySellComponent.vue:381]: remove unused code.

**Review by gamalielhere — CHANGES_REQUESTED** [review]

**Review by gamalielhere — CHANGES_REQUESTED** [review]

**Review by hsurf22 — APPROVED** [review]

**Review by gamalielhere — APPROVED** [review]

**Pull request merged**